qcodes_contrib_drivers/drivers/Attocube/AttocubeController.py

```diff
@@ -0,0 +1,324 @@
+from typing import List, Dict, Union, Optional, Sequence, Any, Tuple
+import numpy as np
+import visa
+import time
+import logging
+log = logging.getLogger(__name__)
+
+from qcodes.instrument.visa import VisaInstrument
+import qcodes.utils.validators as vals
+
+class AttocubeController(VisaInstrument):
+    """Base class for Attocube controller instrument.
+
+    This driver "low temperature" and "room temperature" modes, with different
+    voltage limits to protect the Attocubes.
+
+    I have found the Attocube serial communication to be kind of sketchy,
+    so ask_raw() is overridden try to make communication more robust. YMMV.
+
+    Args:
+        atto_config (dict): Configuration dict (example below).
+        temp (str): 'LT' or 'RT', depending on whether the attocubes are at room temp. or low temp.
```

Comment:
Could we please remove the types from the docstring? They already defined in the signature.

Reply:
Good point, done here: https://github.com/QCoDeS/Qcodes_contrib_drivers/pull/38/commits/4832b40ef5d657b3bd72652ed08dd37279402732